app/api/v2/managers/contact_api_manager.py

```diff
@@ -0,0 +1,13 @@
+from app.api.v2.managers.base_api_manager import BaseApiManager
+from app.api.v2.responses import JsonHttpNotFound
+
+
+class ContactApiManager(BaseApiManager):
+    def __init__(self, data_svc, file_svc, contact_svc):
+        super().__init__(data_svc=data_svc, file_svc=file_svc)
+        self.contact_svc = contact_svc
+
+    def get_contact_report(self, contact: str = None):
+        if contact in self.contact_svc.report:
+            return self.contact_svc.report.get(contact)
```

Comment:
what if we do `contact.upper()` in case the user requests with lowercase in the URL?